### ISSUE TITLE
set up method in connection.py to handle request problems

### ISSUE THREAD

**Issue by odanoburu** [labFGV/bradata#17]
every submodule should have some information about it, such as which url's it scrapes/downloads data from and what's the website's name. using this information and the submodule's name, one could create a package-wise method to warn users when links become stale, such as:
```python
print("""the request failed with code {}:
{}
the {} server may be down, or it may have changed its address or architecture. 
please report the latter to the maintainers. 
you can check if the website is online at {}""".format((req.status_code, req.text, website.name, website.url)
```

obs: formatting was done to make it easier to read.

**Comment by JoaoCarabetta** [comment]
This can be a general warning. We can put it on connection.py and call it when something goes really wrong

**Comment by odanoburu** [comment]
exactly! there's already a stub there, it just needs a little more work

**Comment by odanoburu** [comment]
@JoaoCarabetta , I checked your code and how you used the `connection.py` module. maybe we should change a little how it works? for instance, at some point you'd import the method from the class instead of instantiating it; that's because currently there's no need for a class -- it only has one method and it's not the `__init__` one!

so we can make it a single method, or give it more useful methods. (not sure which would be better). we could improve the point mentioned in this issue too. any other ideas??

**Comment by JoaoCarabetta** [comment]
As we have talked before, this may not be needed as a global method. 

We may use requests as a library to request data from the internet and if someone needs something more, he or she can build a exclusive connection module.

What do you think?

**Comment by odanoburu** [comment]
okay, I agree. so should we have a guideline asking for people to use requests (unless it's **really** necessary to use something else)? just so we don't have each contributor using a different library and our dependencies only growing..

can you delete from the utils.py what we're not using?
